msteams-platform/concepts/app-fundamentals-overview.md

```diff
@@ -76,10 +76,10 @@ The following table includes Teams features and availability for GCC, GCC-High,
 | Teams owned apps as in internally developed apps | ✔️ App is enabled if it has GCC. | ✔️ App is enabled if it has GCC-High. | ✔️ App is enabled if it has DOD. |
 | Microsoft apps | ✔️ Microsoft apps compliant with GCC. | ✔️ Microsoft apps compliant with GCC-High. | ✔️ Microsoft apps compliant with DOD. |
 | 3P or third-party apps | ✔️ Third-party apps are available. Disabled by default and tenant admin use their own discretion to enable it. | ❌ | ❌ |
-| Bots | ✔️ | ✔️ | ❌ |
+| Bots | ✔️ | ✔️ | ✔️ |
 | Custom or Lob tab apps |  ✔️ | ✔️ | ✔️ |
 | Sideloading apps | ✔️ | ❌ | ❌ |
-| Custom or Lob bots | ✔️ | ✔️ | ❌ |
+| Custom or Lob bots | ✔️ | ✔️ | ✔️ |
```

Comment:
We should put LOB as a separate entry (right now we have LOB with tabs, with bots etc).

Messaging extensions will be available in DOD.

Reply:
We have captured 2 categories here:

How are apps accessible in Gov environments:
1P, 3P - Store apps, LOB, Side loaded.

What capabilities are available in these environments:
Tab, Bots, MEs, Connectors. We can either have a separate table for these or change the order to contain all the apps first.

Reply:
This seems like a structuring comment for the table. We have types of apps and then capabilities - maybe we could structure the table to show the apps supported first and then the capabilities: 
Apps - 1P, 3P, LOB, Sideloading
Capabilities - Tab, Bots, MEs, Connectors, apps in meetings, etc.